Makefile.PL

```diff
@@ -33,7 +30,7 @@ my %WriteMakefileArgs = (
     "Test::More" => 0,
     "utf8" => 0
   },
-  "VERSION" => "0.25",
+  "VERSION" => "0.26",
```

Comment:
Other than this, the rest of the changes don't seem to be necessary; probably let upstream update this.

Reply:
Should I remove this too? https://github.com/aminmarashi/Locale-Country-Multilingual/pull/1/files#diff-c112bb3542e98308d12d5ecb10a67abcR5

Reply:
The changes blurb is actually fine, though the `0.26 ...` line can be removed (so whoever runs the `dzil release` can pick the lines via `{{NEXT}}`...